vector/src/main/AndroidManifest.xml

```diff
@@ -417,6 +420,22 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/sdk_provider_paths" />
         </provider>
+
+        <!-- Temporary fix for Android 12. android:exported has to be explicitly set when targeting Android 12
+        Do it for services coming from dependencies - BEGIN -->
```

Comment:
:100: hopefully the libraries will update their targets soonish~ (especially androidx)

Reply:
Yes. Jitsi lib has not been upgrading since a while, and it requires some manual steps. See https://github.com/vector-im/element-android/blob/main/docs/jitsi.md if you are curious (I know you are :) )